Index: llvm-project/clang/bindings/python/clang/cindex.py
===================================================================
--- llvm-project.orig/clang/bindings/python/clang/cindex.py
+++ llvm-project/clang/bindings/python/clang/cindex.py
@@ -82,6 +82,7 @@
     py_object,
 )

+import ctypes.util
 import os
 import sys
 from enum import Enum
@@ -4357,7 +4358,7 @@
         if name == "Darwin":
             file = "libclang.dylib"
         elif name == "Windows":
-            file = "libclang.dll"
+            file = "libclang-13.dll"
         else:
             file = "libclang.so"

@@ -4370,6 +4371,17 @@
     def get_cindex_library(self) -> CDLL:
         try:
             library = cdll.LoadLibrary(self.get_filename())
+        except OSError:
+            pass
+        else:
+            return library
+        t = ctypes.util.find_library(self.get_filename())
+        if t:
+            return cdll.LoadLibrary(t)
+
+        # redo for getting proper system-error message
+        try:
+            library = cdll.LoadLibrary(self.get_filename())
         except OSError as e:
             msg = (
                 str(e) + ". To provide a path to libclang use "
