Index: llvm-project/clang/bindings/python/clang/cindex.py
===================================================================
--- llvm-project.orig/clang/bindings/python/clang/cindex.py	2024-09-16 10:41:00.459562394 -0500
+++ llvm-project/clang/bindings/python/clang/cindex.py	2024-09-16 10:45:29.051291466 -0500
@@ -63,6 +63,7 @@
 # o implement additional SourceLocation, SourceRange, and File methods.
 
 from ctypes import *
+import ctypes.util
 
 import clang.enumerations
 
@@ -3935,7 +3936,7 @@
         if name == "Darwin":
             file = "libclang.dylib"
         elif name == "Windows":
-            file = "libclang.dll"
+            file = "libclang-13.dll"
         else:
             file = "libclang.so"
 
@@ -3947,6 +3948,17 @@
     def get_cindex_library(self):
         try:
             library = cdll.LoadLibrary(self.get_filename())
+        except OSError:
+            pass
+        else:
+            return library
+        t = ctypes.util.find_library(self.get_filename())
+        if t:
+            return cdll.LoadLibrary(t)
+
+        # redo for getting proper system-error message
+        try:
+            library = cdll.LoadLibrary(self.get_filename())
         except OSError as e:
             msg = (
                 str(e) + ". To provide a path to libclang use "
